apps/web/screens/NewsList.tsx

```diff
@@ -127,41 +144,49 @@ const NewsList: Screen<NewsListProps> = ({
             <Link href={makePath()}>Ísland.is</Link>
             <Link href={makePath('news')}>Fréttir og tilkynningar</Link>
           </Breadcrumbs>
-          <Hidden below="lg">
-            <Typography variant="h1" as="h1">
-              {selectedYear}
-            </Typography>
-          </Hidden>
+          {selectedYear && (
+            <Hidden below="lg">
+              <Typography variant="h1" as="h1">
+                {selectedYear}
+              </Typography>
+            </Hidden>
+          )}
 
           <GridRow>
             <GridColumn hideAbove="sm" span="12/12" paddingBottom={1}>
               <Select
                 label="Ár"
-                placeholder="Ár"
-                value={yearOptions.find(
-                  (o) => o.value === selectedYear.toString(),
-                )}
+                placeholder={allYears}
+                value={yearOptions.find((o) => {
+                  o.value ===
+                    (selectedYear ? selectedYear.toString() : allYears)
+                })}
                 options={yearOptions}
-                onChange={({ value }: Option) => Router.push(makeHref(value))}
-                name="year"
-              />
-            </GridColumn>
-            <GridColumn hideAbove="sm" span="12/12">
-              <Select
-                label="Mánuður"
-                placeholder="Allt árið"
-                value={monthOptions.find((o) => o.value === selectedMonth)}
-                options={monthOptions}
                 onChange={({ value }: Option) =>
-                  Router.push(makeHref(selectedYear, value))
+                  Router.push(makeHref(value === selectedYear ? value : null))
                 }
-                name="month"
+                name="year"
               />
             </GridColumn>
+            {selectedYear && (
+              <GridColumn hideAbove="sm" span="12/12">
```

Comment:
there should probably never be a need to use span=12/12, here it is only required to counter the the negative margin from row and it looks like row is unnecessary here as the columns are not used. You don't need to change this I just wanted to point that out :)